packages/palette/src/elements/Button/Button.tsx

```diff
@@ -91,123 +77,32 @@ const getSize = ({
   }
 }
 
-const getVariant = ({ variant }: Pick<ButtonProps, "variant">) => {
-  switch (variant) {
-    case "primaryBlack":
-      return css`
-        ${props => {
-          const { colors } = props.theme
-
-          return `
-              background-color: ${colors.black100};
-              border-color: ${colors.black100};
-              color: ${colors.white100};
-
-              @media ${themeProps.mediaQueries.hover} {
-                &:hover {
-                  background-color: ${colors.purple100};
-                  border-color: ${colors.purple100};
-                  color: ${colors.white100};
-                }
-              }
-            `
-        }};
-      `
-    case "primaryWhite":
-      return css`
-        ${props => {
-          const { colors } = props.theme
-
-          return `
-              background-color: ${colors.white100};
-              border-color: ${colors.white100};
-              color: ${colors.black100};
-
-              @media ${themeProps.mediaQueries.hover} {
-                &:hover {
-                  background-color: ${colors.purple100};
-                  border-color: ${colors.purple100};
-                  color: ${colors.white100};
-                }
-              }
-            `
-        }};
-      `
-    case "secondaryGray":
-      return css`
-        ${props => {
-          const { colors } = props.theme
-
-          return `
-              background-color: ${colors.black10};
-              border-color: ${colors.black10};
-              color: ${colors.black100};
-
-              @media ${themeProps.mediaQueries.hover} {
-                &:hover {
-                  background-color: ${colors.black30};
-                  border-color: ${colors.black30};
-                  color: ${colors.black100};
-                }
-              }
-            `
-        }};
-      `
-    case "secondaryOutline":
-      return css`
-        ${props => {
-          const { colors } = props.theme
-          return `
-              background-color: ${colors.white100};
-              border-color: ${colors.black10};
-              color: ${colors.black100};
-
-              @media ${themeProps.mediaQueries.hover} {
-                &:hover {
-                  background-color: ${colors.white100};
-                  border-color: ${colors.black100};
-                  color: ${colors.black100};
-                }
-              }
-            `
-        }};
-      `
-    case "noOutline":
-      return css`
-        ${props => {
-          const { colors } = props.theme
-          return `
-              background-color: transparent;
-              border-color: transparent;
-              color: ${colors.black100};
-            `
-        }};
-      `
-    default:
-  }
-}
-
```

Comment:
👋

Reply:
Awesome update with the `variant` -- I've never noticed that fn before. Also didn't know that would could handle responsivity using object notation.